src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -2,19 +2,33 @@ package racingcar.domain
 
 import racingcar.domain.carracer.CarRacer
 
-private const val DEFAULT_MOVEMENT = 0
-private const val MOVING_BENCHMARK = 4
+class RacingCar(
+    name: Name,
+    movement: Movement,
+    private val carRacer: CarRacer,
+) {
+    private var _name: Name = name
+    private var _movement: Movement = movement
+    val name: String
+        get() = _name.value
+    val movement: Int
+        get() = _movement.value
 
-class RacingCar(private val carRacer: CarRacer) {
+    constructor(carRacer: CarRacer) : this(
+        name = Name(),
+        movement = Movement(),
+        carRacer,
```

Comment:
named arguments 를 전부 동일하게 적용해주시는게 유지보수하기 좋을것 같습니다.

```suggestion
        name = Name(),
        movement = Movement(),
        carRacer = carRacer,
```